content/index.de.md

```diff
@@ -0,0 +1,248 @@
+  <p>Entwickelt von der Arbeitsgruppe für Bildung und Öffentlichkeitsarbeit. (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Zusammenfassung" class="full" %}
+{:/}
+
+Wenn Webseiten und Webwerkzeuge richtig gestaltet und programmiert sind, können Menschen mit Behinderungen sie nutzen. Derzeit werden jedoch viele Seiten und Werkzeuge mit Barrieren entwickelt, die ihre Nutzung für manche Menschen schwer oder unmöglich machen.
+
+Das Web barrierefrei zu machen dient Einzelnen, Unternehmen sowie der Gesellschaft. Internationale Webstandards legen fest, was für Barrierefreiheit erforderlich ist.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Inhaltsverzeichnis" class="full" %}
+{:/}
+
+-   Das Inhaltsverzeichnis wurde automatisch erstellt.
+{:toc}
+
+<span class="box-h box-h-simple box-h-full">Verwandte Quelle</span><br>
+{% include video-link.html title="Videoeinführung in das barrierefreie Web und W3C-Standards <em>(4 Minuten)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Barrierefreiheit im Kontext {#context}
+
+<blockquote class="pull">
+  <p>Die Stärke des Webs liegt in dessen Universalität.<br>
+    Der Zugriff aller Menschen, unabhängig von Behinderungen, ist ein unabdingbarer Bestandteil.</p>
+  <footer><cite>Tim Berners-Lee, Leiter des W3C und Erfinder des World Wide Web</cite></footer>
+</blockquote>
+
+Das Web ist grundlegend so gestaltet, dass es für alle Menschen unabhängig von deren Hardware, Software, Sprache, Ort oder Fähigkeiten funktioniert. Wenn das Web dieses Ziel erreicht, ist es für Menschen mit unterschiedlichem Hör-, Bewegungs-, Seh- und Denkvermögen zugänglich.
+
+Der Einfluss von Behinderungen ist im Web anders, weil das Web Barrieren in der Kommunikation und Interaktion beseitigt, denen Menschen in der physischen Welt begegnen. Wenn Webseiten, Anwendungen, Technologien oder Werkzeuge allerdings schlecht gestaltet sind, können sie Barrieren erzeugen, die Menschen an der Nutzung des Webs hindern.
+
+**Barrierefreitheit ist essenziell für Entwickler:innen und Organisationen, die hochwertige Webseiten und Webwerkzeuge erstellen und Menschen nicht an der Nutzung ihrer Produkte und Dienstleistungen hindern wollen.**
+
+
+## Was ist das barrierefreie Web {#what}
+
+Barrierefreies Web bedeutet, dass Webseiten, Werkzeuge und Technologien so gestaltet sind, dass Menschen mit Behinderungen sie nutzen können. Genauer gesagt können Menschen
+
+- das Web wahrnehmen, verstehen, navigieren sowie mit ihm interagieren
+- zum Web beitragen
+
+Barrierefreies Web schließt alle Behinderungen ein, die den Zugang zum Web beeinträchtigen, einschließlich
+
+- kognitive,
```

Comment:
The order of the items here are different than in the original version. For consistency, please keep the original order.

Reply:
The reason, I changed the order is because some of the forms of disability are translated as a single noun, s.a. "Hörbehinderung", "Sehbehinderung", while others are translated as adjective + "Behinderung", s.a. "kognitive Behinderung", "körperliche Behinderung". Putting the items in the original order sounds a little odd to me: "Hör-, kognitive, neurologische, körperliche, Sprach- und Sehbehinderungen." But I don't mind changing it, if the order is important.

Reply:
@alexejrotar I would say either keep the original order or order items alphabetically (since this explains the original order)